src/main/java/org/sagebionetworks/bridge/exporter/synapse/SynapseStatusTableHelper.java

```diff
@@ -0,0 +1,141 @@
+            InterruptedException, SynapseException {
+        // Does the table already exist? If not, create it.
+        String synapseTableId = manager.getSynapseTableIdFromDdb(task, SynapseHelper.DDB_TABLE_SYNAPSE_META_TABLES,
+                SynapseHelper.DDB_KEY_TABLE_NAME, getStatusTableName(studyId));
+        if (synapseTableId == null) {
+            synapseTableId = createStatusTable(task, studyId);
+        }
+
+        // Table definitely exists now. Write status with this internal helper.
+        writeStatus(synapseTableId, task, studyId);
+    }
+
+    // Helper method that abstracts away the status table name, which is always "[studyId]-status".
+    private String getStatusTableName(String studyId) {
+        return studyId + "-status";
+    }
+
+    // Helper method to create the status table. It sets up the columns, then calls through Synapse Helper.
+    private String createStatusTable(ExportTask task, String studyId) throws BridgeExporterException,
+            SynapseException {
+        // Delegate table creation to SynapseHelper.
+        long dataAccessTeamId = manager.getDataAccessTeamIdForStudy(studyId);
+        long principalId = manager.getSynapsePrincipalId();
+        String projectId = manager.getSynapseProjectIdForStudyAndTask(studyId, task);
+        String tableName = getStatusTableName(studyId);
+        String synapseTableId = synapseHelper.createTableWithColumnsAndAcls(COLUMN_LIST, dataAccessTeamId, principalId,
+                projectId, tableName);
+
+        // write back to DDB table
+        manager.setSynapseTableIdToDdb(task, SynapseHelper.DDB_TABLE_SYNAPSE_META_TABLES,
+                SynapseHelper.DDB_KEY_TABLE_NAME, getStatusTableName(studyId), synapseTableId);
+
+        return synapseTableId;
+    }
+
+    // Helper method to write the actual status. It creates a partial row set, then calls through to the Synapse
+    // Helper to write the row set.
+    private void writeStatus(String synapseTableId, ExportTask task, String studyId) throws BridgeExporterException,
+            InterruptedException, SynapseException {
+        // The row set map is one column ID, so we need to get the column model from the table.
+        List<ColumnModel> columnList = synapseHelper.getColumnModelsForTableWithRetry(synapseTableId);
+        if (columnList.size() != 1) {
+            throw new BridgeExporterException("Wrong number of columns in status table for study " + studyId +
+                    ", expected 1 column, got " + columnList.size() + " columns");
+        }
+
+        // We know there is exactly one column in the status table.
+        ColumnModel column = columnList.get(0);
+        String colName = column.getName();
+        if (!COLUMN_NAME_UPLOAD_DATE.equals(colName)) {
+            throw new BridgeExporterException("Wrong column in status table for study " + studyId + ", expected " +
+                    COLUMN_NAME_UPLOAD_DATE + ", got " + colName);
+        }
```

Comment:
We intentionally separate Bridge-EX tables from researcher tables. This is to prevent researchers from modifying the original data (accidentally or intentionally). This follows the same model as all the other tables Bridge-EX has created.

There's also not really any point for researchers to add information to this table. This table is intended entirely to signal automated processes that Bridge-EX has finished running for the day. If researchers need to add their own data about a day's worth of data, there's nothing stopping them from creating their own tables.